### ISSUE TITLE
Dark theme + support for theme & plugin override

### ISSUE THREAD

**Issue by rykdesjardins** [narcitymedia/lilium-cms#414]
**What section of the CMS does it affect?**
Interface, preferences

**Is your feature fixing a problem? Please describe.**
At night, Lilium pretty much makes your eyes bleed. A simple LESS file with "dark variables" would do the trick. The dark theme would be compiled as a separate stylesheet and would be compiled using the dark variables instead of the light ones. Since all constants are isolated in a file, it should be a pretty quick job. 

The rest is a big `grep` + `sed` job to find border colors and other non-constant colors. 

Since plugins and themes also require styling, they could provide a dark version of their stylesheets as `filename.dark.css`.

Themes can override TinyMCE's default stylesheet by providing a file at the following location relative to the markup directory : `/compiled/theme/tinymce.css`. The theme could also provide `/compiled/theme/tinymce.dark.css`, which would be downloaded if the dark theme is enabled by the user. 

**Steps needed to access / experience your new feature**
1 - Activate dark theme from the preferences
2 - Save your eyes :eyes:
